cypress/page/shipping-step.page.ts

```diff
@@ -0,0 +1,21 @@
+class ShippingstepPage{
+
+    private checkBox: string;
+    private carrier: string;
```

Comment:
could you add to the attributes of each page object the type of element we are interacting with?